mlflow/server/js/src/experiment-tracking/components/App.css

```diff
@@ -92,6 +92,22 @@ div.docs {
   color: #ffffff;
 }
 
+.header-nav-link-experiments {
+  display: inline-block;
+  height: 39px;
+}
+
+.header-nav-link-models {
+  display: inline-block;
+  margin-left: 24px;
+  height: 39px;
+}
+
+.experiments, .models {
+  display: inline-block;
```

Comment:
`inline-block` mandates a font-size related gap between divs. Our current convention is to use `display: flex` on the parent div to get the block-level element horizontal laid out.